Tests/SonarScanner.MSBuild.Common.Test/RegularExpressions/WildcardPatternMatcherTest.cs

```diff
@@ -28,7 +28,6 @@ public class WildcardPatternMatcherTest
     /// <summary>
     /// Based on https://github.com/SonarSource/sonar-plugin-api/blob/master/plugin-api/src/test/java/org/sonar/api/utils/WildcardPatternTest.java.
     /// </summary>
-    [TestCategory(TestCategories.NoUnixNeedsReview)]
```

Comment:
Please duplicate this test. One for Windows and one for Linux/MacOS. Change the input by replacing the `/` on Windows with `\`.